example-search.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
This would be better in the documentation (with example values).

Reply:
Maybe this should be named `default-search.json` (given the intent explained in the Dockerfile comments)

Reply:
I'm now using `constants.py` which can be used to retrieve application constants.